SparkPost/2web2ui
# SA-892: Sort by injection counts

### What Changed
 - Added sortable column for injectionand engagement recency
 - Added `order_by` mapping for keys

### How To Test
 - Go to `/signals/engagement`
 - Verify that the injection column is sortable

### To Do
- [x] Check if spam traps will have filter

### Notes
 - Spam traps endpoints `order_by` doesn't currently seem to have a way to order by injections. Total injections are also currently calculated on the front end
